bucket_map/src/bucket_storage.rs

```diff
@@ -112,6 +113,7 @@ impl BucketStorage {
         elem_size: u64,
         max_search: MaxSearch,
         stats: Arc<BucketStats>,
+        count: Arc<AtomicU64>,
```

Comment:
Can this be standardized to `used`, since that's what the field in the struct is named?